storybook/storybook-react/src/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -0,0 +1,289 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid, Heading, Link, MegaMenu, UnorderedList } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Navigation/Mega Menu',
+  component: MegaMenu,
+  parameters: {
+    layout: 'fullscreen',
+  },
```

Comment:
Let’s hide `children` from the controls table.

```suggestion
  },
  argTypes: {
    children: {
      table: {
        disable: true,
      },
    },
  },
```

Reply:
Should we always do this when children are used?

Reply:
Yes, I think so.